stroyprombeton/settings/base.py

```diff
@@ -254,68 +256,73 @@
     },
 }
 
-CUSTOM_PAGES = {
-    'category-tree': {
-        'slug': 'gbi',
-        'title': 'Каталог товаров',
-        'name': 'Все категории',
-        'menu_title': 'Каталог',
-    },
-    'client-feedbacks': {
-        'slug': 'client-feedbacks',
-        'name': 'Отзывы',
-    },
-    'index': {
-        'slug': '',
-        'title': (
-            'Завод ЖБИ «СТК-ПромБетон» | Производство ЖБИ в Санкт-Петербурге,'
-            ' железобетонные изделия СПб'
-        ),
-        'name': 'Завод железобетонных изделий «СТК-Промбетон»',
-        'menu_title': 'Главная',
-    },
-    'news': {
-        'slug': 'news',
-        'name': 'Новости компании',
-        'title': 'Завод ЖБИ «СТК-ПромБетон»',
-        'menu_title': 'Новости компании'
-    },
-    'order': {
-        'slug': 'order',
-        'title': 'Корзина Интернет-магазин СТК-ПромБетон',
-        'name': 'Оформление заказа',
-    },
-    'order-price': {
-        'slug': 'order-price',
-        'title': 'Оформление запроса на прайс ЖБИ',
-        'name': 'Оформление запроса на прайс ЖБИ',
-        'keywords': 'прайс лист жби',
-        'description': 'Оформить заявку на прайслист ЖБИ завода СТК-Промбетон',
-    },
-    'order-success': {
-        'slug': 'order-success',
-        'title': 'Спасибо за Ваш заказ',
-        'name': 'Заказ принят',
-    },
-    'regions': {
-        'slug': 'regions',
-        'name': 'Регионы, в которые поставлялась продукция СТК-ПромБетон',
-    },
-    'search': {
-        'slug': 'search',
-        'title': 'Результаты поиска',
-        'name': 'Результаты поиска',
-    },
-    'sitemap': {
-        'slug': 'sitemap',
-        'h1': 'Карта сайта',
-        'name': 'Карта сайта',
-    },
-    'order-drawing': {
-        'slug': 'order-drawing',
-        'name': 'Оформление запроса на изготовление по индивидуальным чертежам',
-    },
-}
+with open(os.path.join(TEMPLATE_DIR, 'robots.txt')) as robots_file:
```

Comment:
it's not good to wrap the whole big structure in `with` statement.
Only it's one small part uses rotobts_file.
Let's get robots_file with function